README.md

```diff
@@ -13,6 +12,33 @@ Control plane clusters alert rules are maintained by the same team, but are kept
 different
 [repository](https://gitlab.cee.redhat.com/service/app-interface/-/tree/master/resources/stonesoup/argocd-control-plane/monitoring).
 
+The different alerting rules in this repository are:
+
+* **Alert Rule ControllerReconciliationErrors**
+
+    The alert indicates that a controller's reconciliation failure ratio is higher than the
+    allowed threshold. Controller reconciliation is the process of synchronizing resources over which a controller is watching with the business logic specifications the controller implements. Controller reconciliation errors happen when a controller fails enforcing a desired state (the specifications) over a resource it's watching. This alert is triggered only for controllers under RHTAP service namespaces. Namely, it will not be triggered for neither controllers used for OpenShift's built-in functionality nor for controllers in tenant namespaces.
```

Comment:
Let's keep the lines of text at a fixed length (same for the rest of the document):
```suggestion
    The alert indicates that a controller's reconciliation failure ratio is higher than
    the allowed threshold. Controller reconciliation is the process of synchronizing
    resources over which a controller is watching with the business logic specifications
    the controller implements. Controller reconciliation errors happen when a controller
    fails enforcing a desired state (the specifications) over a resource it's watching.
    This alert is triggered only for controllers under RHTAP service namespaces. Namely,
    it will not be triggered for neither controllers used for OpenShift's built-in
    functionality nor for controllers in tenant namespaces.
```